### PR TITLE
Print Entering dracut emergency mode instade of Entering emergency mode

### PR DESCRIPTION
**Pull request by wtmpx**
This pull request changes...

## Checklist
- [ ] I have tested it locally
- [ ] I have reviewed and updated any documentation if relevant
- [ ] I am providing new code and test(s) for it

Fixes #

### CODE REVIEW

File: modules.d/35network-manager/nm-initrd.service [hunk 1]
```diff
@@ -1,7 +1,7 @@
 [Unit]
 DefaultDependencies=no
-Wants=systemd-udev-settle.service
-After=systemd-udev-settle.service
+Wants=systemd-udev-trigger.service
+After=systemd-udev-trigger.service
```
Comment (haraldh) [inline modules.d/35network-manager/nm-initrd.service:4]: ???

**Review by haraldh — CHANGES_REQUESTED** [review]

File: modules.d/35network-manager/nm-initrd.service (hunk 1 above)
Comment (wtmpx) [reply modules.d/35network-manager/nm-initrd.service:4]: Sorry, no idea. can you reject above change?

Comment (haraldh) [reply modules.d/35network-manager/nm-initrd.service:4]: > Sorry, no idea. can you reject above change?

no.... you have to get your git commit history right

**Comment by haraldh** [comment]
no we have 3 commits.. maybe squash them :)

**Comment by johannbg** [comment]
I'm nack-ing this PR beside it being to messy to merge as well as if you think that the emergency mode msg is to similar to systemd then make a PR against systemd to change it's message to clarify that you are in it since dracut emergency mode pre-dates systemd. That said it does not matter if the emergency mode msg is the same or similar ( which is being done for consistency ) since the user has been dropped into emergency mode in which it's expected that individual dealing with the emergency knows what he or she is doing thus has no problem figuring it out at which stage of the OS stack, the OS entered it...

**Comment by wtmpx** [comment]
> if you think that the emergency mode msg is to similar to systemd then make a PR against systemd to change it's message to clarify that you are in it since dracut emergency mode pre-dates systemd.

I think for normal user when they need to explain where the system stop booting a clear message would be much easier/clear.

**Pull request closed without merging**